torchgeo/datasets/bigearthnet.py

```diff
@@ -504,3 +509,71 @@ def _extract(self, filepath: str) -> None:
         """
         if not filepath.endswith(".csv"):
             extract_archive(filepath)
+
+    def _onehot_labels_to_names(
+        self, label_mask: np.ndarray  # type: ignore[type-arg]
+    ) -> List[str]:
+        """Gets a list of class names given a label mask.
+
+        Args:
+            label_mask: a boolean mask corresponding to a set of labels or predictions
+
+        Returns
+            a list of class names corresponding to the input mask
+        """
+        labels = []
+        for i, mask in enumerate(label_mask):
+            if mask:
+                labels.append(self.class_sets[self.num_classes][i])
+        return labels
+
+    def plot(
+        self,
+        sample: Dict[str, Tensor],
+        show_titles: bool = True,
+        suptitle: Optional[str] = None,
+    ) -> plt.Figure:
+        """Plot a sample from the dataset.
+
+        Args:
+            sample: a sample returned by :meth:`__getitem__`
+            show_titles: flag indicating whether to show titles above each panel
+            suptitle: optional string to use as a suptitle
+
+        Returns:
+            a matplotlib Figure with the rendered sample
+
+        Raises:
+            ValueError: if ``self.bands`` is "s1"
+
+        .. versionadded:: 0.2
+        """
+        if self.bands == "s2":
+            image = np.rollaxis(sample["image"][[3, 2, 1]].numpy(), 0, 3)
+        elif self.bands == "all":
+            image = np.rollaxis(sample["image"][[5, 4, 3]].numpy(), 0, 3)
+        else:
+            raise ValueError("The s1 band set does not contain RGB bands")
```

Comment:
I still think we can and should plot s1 as grayscale if someone wants to display the data

Reply:
kk this is easy enough and makes the test look nicer (is this why you are asking for this?)

Reply:
Not actually asking because of test simplification, just think there's no reason not to support it.